python/gui/auto_generated/raster/qgsrasterlayerproperties.sip.in

```diff
@@ -0,0 +1,59 @@
+/************************************************************************
+ * This file has been generated automatically from                      *
+ *                                                                      *
+ * src/gui/raster/qgsrasterlayerproperties.h                            *
+ *                                                                      *
+ * Do not edit manually ! Edit header and run scripts/sipify.pl again   *
+ ************************************************************************/
+
+
+
+
+
+
+
+class QgsRasterLayerProperties : QgsOptionsDialogBase
+{
+%Docstring
+Property sheet for a raster map layer
+
+.. versionadded:: 3.10
+%End
+
+%TypeHeaderCode
+#include "qgsrasterlayerproperties.h"
+%End
+  public:
+
+    enum StyleType
```

Comment:
Hide this from the python bindings too, by wrapping the definition in the .h file with

```
#ifndef SIPRUN

...
#endif
```